charts/ratify/values.yaml

```diff
@@ -43,5 +46,7 @@ akvCertConfig:
   clientId:
```

Comment:
azureWorkloadIdentity could be used for both csi driver Keyvault auth and oras auth. Should we move 'azureWorkloadIdentity' outside of oras auth, so we could have a single clientId property?  Current akvCertConfig.clientId is also the azwi clientId.

Reply:
Good to know - will change this as discussed in the community meeting.

Reply:
This should be ready for further review now